packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useState } from 'react';
+import { KeyboardAvoidingView, View } from 'react-native';
+
+import { styles } from './style';
+import PrimaryButton from '../../components/PrimaryButton/PrimaryButton';
+import InputBox from '../../components/InputBox/InputBox';
+
+import { useArbiterInfo } from '../../context/ArbiterInfoContext';
+import Sheet from '../../components/Sheet/Sheet';
+import { ScrollView } from 'react-native-gesture-handler';
+import IconButton from '../../components/IconButton/IconButton';
+import PrimaryText, {
+  FontWeight,
+} from '../../components/PrimaryText/PrimaryText';
+import Link from '../../components/Link/Link';
+import { torneloUrl } from '../../util/env';
+import { ArbiterInfo } from '../../types/ArbiterInfoState';
+import { useError } from '../../context/ErrorContext';
+import { validUrl } from '../../util/url';
+import { colours } from '../../style/colour';
+
+export type EditCurrentEventParams = {
+  display: boolean;
+  dismiss: () => void;
+};
+const EditCurrentEvent: React.FC<EditCurrentEventParams> = ({
+  dismiss,
+  display,
+}) => {
+  const [arbiterInfo, setArbiterInfo] = useArbiterInfo();
+  const [, showError] = useError();
+
+  const [inScanQRMode, setInScanQRMode] = useState(true);
+
+  const [arbiterPin, setArbiterPin] = useState(arbiterInfo?.pin ?? '');
+  const [pgnUrl, setpgnUrl] = useState(arbiterInfo?.broadcastUrl ?? '');
+  const [arbiterEmailSecret, setArbiterEmailSecret] = useState(
+    arbiterInfo?.arbiterToken ?? '',
+  );
+  const [divisionId, setdivisionId] = useState(arbiterInfo?.divisionId ?? '');
+  const [arbiterId, setArbiterId] = useState(arbiterInfo?.userId ?? '');
+  const [eventName, seteventName] = useState(arbiterInfo?.eventName ?? '');
```

Comment:
is there a reason setpgnUrl, setdivisionId, and seteventName don't have the second word capitalized?